### ISSUE TITLE
Cut a new release

### ISSUE THREAD

**Issue by jaraco** [pypa/build#166]
In #158, I encountered an issue that's been fixed in master. It would be nice to have a release that includes that fix.

**Comment by gaborbernat** [comment]
At the moment only @FFY00 can do this 🤷 But we could do release ``0.0.5``.

**Comment by FFY00** [comment]
I would really like #132 to be fixed before a new release, which is not *that* straight forward because of the tests. But since there are major issues with the current one, I think we can defer it.

**Comment by pganssle** [comment]
Presumably we should make the version `0.1.0`? We've got major new features and in fact backwards-compatibility breaks.

Also, I think that at this stage in the project there's no reason to delay releases waiting on new features or bugfixes, as long as the version on master is an improvement on the released version.

**Comment by FFY00** [comment]
I am fine with that.

**Comment by gaborbernat** [comment]
Agreed with @pganssle, we should release as often as possible until we reach 1.0.

**Comment by FFY00** [comment]
Just releases 0.1.0 :blush:
